src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/DefendantResponseNotificationService.java

```diff
@@ -99,13 +99,16 @@ public void notifyClaimant(
         Response response = claim.getResponse().orElseThrow(IllegalStateException::new);
         Map<String, String> parameters = aggregateParams(claim, response);
 
-        String emailTemplate = getClaimantEmailTemplate(response);
+        String emailTemplate = getClaimantEmailTemplate(response, ResponseHelper.admissionResponse(claim));
```

Comment:
I know this is not your code, but `ResponseHelper.admissionResponse` should take argument of type `Response` and NOT `Claim`.

You should't be passing this extra boolean parameter here. Rather have check on line 109 like:
Replace
```
 if (isAdmissionResponse) {
```
with 
```
 if (ResponseHelper.admissionResponse(response)) {
...
}
```